lib/reciter.rb

```diff
@@ -15,4 +15,43 @@ def read
     "This is the horse and the hound and the horn that belonged to the farmer sowing his corn that kept the rooster that crowed in the morn that woke the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n"
   end
 
+  def recite_random_rhyme(random_indices=generate_random_indices)
+    phrases = [
+      "the house that Jack built",
+      "the malt that lay in",
+      "the rat that ate",
+      "the cat that killed",
+      "the dog that worried",
+      "the cow with the crumpled horn that tossed",
+      "the maiden all forlorn that milked",
+      "the man all tattered and torn that kissed",
+      "the priest all shaven and shorn that married",
+      "the rooster that crowed in the morn that woke",
+      "the farmer sowing his corn that kept",
+      "the horse and the hound and the horn that belonged to",
+    ]
+    continous_phrase = ""
+    random_rhyme = ""
+
+    for random_index in random_indices
+      continous_phrase = phrases[random_index].strip + " " + continous_phrase
+      random_rhyme += "This is " + continous_phrase.strip + ".\n"
+    end
+
+    random_rhyme
+  end
+
+  def generate_random_indices
```

Comment:
You can probably make this simpler using the Random library: https://ruby-doc.org/core-3.0.1/Random.html